code/mob/living/critter/ai/generic_critter.dm

```diff
@@ -116,7 +118,6 @@
 
 /datum/aiTask/sequence/goalbased/critter/attack/fixed_target/New(parentHolder, transTask, atom/fixed_target)
 	..(parentHolder, transTask)
-	add_task(holder.get_instance(/datum/aiTask/succeedable/critter/attack, list(holder)))
```

Comment:
?

Reply:
The parent, /datum/aiTask/sequence/goalbased/critter/attack, does this as well- it's entirely unneeded and caused one extra attack past what seemed to be intended.